docs/topics/search/searching.md

```diff
@@ -290,6 +293,10 @@ And([
 ])
 ```
 
+```{note}
+You must always use either of the 2 patterns to construct phrase search query, that is `Single quotes inside Double quotes ("'")` OR `Double quotes inside Single quotes ('"')` and must not use both of them simultaneously in a single query.
```

Comment:
This doesn't seem correct to me - surely the only reason that the issue of nesting different kinds of quotes arises at all, is that these are artificial examples / test cases where we're passing literal strings to `parse_query_string`, and therefore have to follow Python quoting rules. In real world uses, this will come from user input (e.g. `parse_query_string(request.GET['query'])`), and in that situation the user isn't writing any quotes around the overall query... so there's no reason why that user shouldn't be able to enter a search query like

    "lord of the rings" hobbit 'middle earth'

and in fact we can still test queries like this using Python's triple-quote syntax:

    parse_query_string(""""lord of the rings" hobbit 'middle earth'""")

Reply:
@gasman thanks for the review, what do you think would be the correct correct potential of parsing query? Like to also be able to parse the mixed use of quotes?

Reply:
Sorry for the doubt but isn't the current function also has the same issue? And this enhancement would just increase the scope of the parsing ability?

Reply:
I think the code itself is correct - at least, all the test cases I've tried with mixed quotes, such as `parse_query_string(""" hobbit book:'lord of the rings' author:"jrr tolkien" """)`, seem to be working fine. The problem is that the documentation and tests should not be talking in terms of outer quotes - those are just a detail of the Python syntax we're using when calling `parse_query_string`, and not related to what the `parse_query_string` function is doing.

Reply:
alright in that case i should remove the note docs.

Reply:
removed qoutes centric notes, added extra test cases with mixed phrases